check-licenses/action.yml

```diff
@@ -0,0 +1,34 @@
+name: "Check licenses"
+description: "Check if package licenses in the current environment are among the accepted ones"
+
+inputs:
+  accepted-licenses:
+    description: "List of accepted licenses"
+    default: "MIT BSD Apache PSF HPND"
+    required: false
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+
+    - name: "Check licences of packages"
+      shell: bash
+      run: |
+        echo ${{ inputs.accepted-licenses }}
+        pip install pip-licenses
```

Comment:
Try to execute any Python module using `python -m ...`. This ensures that you use the modules within your environment and not the binaries.

Reply:
Got it!